test/unit/selector.js

```diff
@@ -1,7 +1,7 @@
 QUnit.module( "selector", {
 	beforeEach: function() {
 		this.safari = /\bsafari\b/i.test( navigator.userAgent ) &&
-			!/\bchrome\b/i.test( navigator.userAgent );
+			!/\b(?:headless)?chrome\b/i.test( navigator.userAgent );
```

Comment:
Tests in CI are running on Chrome Headless which has a different user agent, making `this.safari === true`. This is what caused the original test failures; it was just a test issue.